kraken/kubernetes/client.py

```diff
@@ -18,7 +24,9 @@ def list_nodes():
     except ApiException as e:
         logging.error("Exception when calling CoreV1Api->list_node: %s\n" % e)
     for node in ret.items:
-        nodes.append(node.metadata.name)
+        if kraken_node_name != node.metadata.name:
```

Comment:
I don't think it's a good idea to not append the node name to list of nodes if it's kraken_node_name as the purpose of this function is to obtain all the node names and this function might be called at other parts of the program as well.

Reply:
I'll make a separate function similar to the list_nodes just to return all nodes but the one kraken is running on. I think you are going to have to update your get_node function to call it so you don't kill it, correct?

Reply:
Yeah

Reply:
@yashashreesuresh I have created a new function named list_killable_nodes to return the list of nodes that kraken is not running on. Can you please update your node_scenarios request to incorporate this? Thanks!

Reply:
Definitely!